src/odemis/acq/move.py

```diff
@@ -292,24 +297,25 @@ def _doCryoTiltSample(future, stage, focus, rx, rz):
         stage_active_range = stage_md[model.MD_POS_ACTIVE_RANGE]
         focus_deactive = focus_md[model.MD_FAV_POS_DEACTIVE]
         current_pos = stage.position.value
+
         # Check that the stage X,Y,Z are within the limits
         if not _isInRange(current_pos, stage_active_range, {'x', 'y', 'z'}):
             raise ValueError("Current position is out of active range.")
         # To hold the ordered sub moves list to perform the tilting/imaging move
         sub_moves = []
-        # Park focus only if stage rx and rz are equal to 0
+        # Park focus only if stage rx is equal to 0
         # Otherwise stop if it's not already parked
         if not _isNearPosition(focus.position.value, focus_deactive, {'z'}):
-            if _isNearPosition(current_pos,  {'rx': 0, 'rz': 0}, {'rx', 'rz'}):
+            if _isNearPosition(current_pos, {'rx': 0}, {'rx'}):
                 sub_moves.append((focus, focus_deactive))
             else:
                 raise ValueError("Cannot proceed with tilting while focus is not near FAV_POS_DEACTIVE position.")
 
-        if rx == 0 and rz == 0:  # Imaging
+        if rx == 0:  # Imaging
```

Comment:
Now that rz = 0 is not an indication of imaging, left it out this check but kept it as a parameter still for `cryoTiltSample` function.